eng/Version.Details.xml

```diff
@@ -414,5 +414,85 @@
+      <Sha>308c7d0f1fa19bd1e7b768ad13646f5206133cdb</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk.linux-x64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk.linux-arm64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk.win-x64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk.win-arm64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk.osx-x64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk.osx-arm64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Cache.win-x64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Cache.win-arm64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Cache.linux-x64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Cache.linux-arm64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Cache.osx-x64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Runtime.Emscripten.3.1.34.Cache.osx-arm64" Version="9.0.0-preview.4.24175.6">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>2106e0604f52119b7d49577476ece7d51a23a40d</Sha>
+    </Dependency>
```

Comment:
```suggestion
```
This isn't critical for the I believe but the transport package is enough to establish the dependency for darc and these package names will change with the emsdk bump and would require manual updating each time.